protected/controllers/PracticeController.php

```diff
@@ -99,6 +99,9 @@ public function actionCreate($context = null)
     public function performPracticeSave(Contact $contact, Practice $practice, Address $address, $isAjax = false)
     {
         $action = $practice->isNewRecord ? 'add' : 'edit';
+        if (!$practice->code) {
+            $practice->code = 'CERA'; // This will be the same for ALL practices added through the frontend. But only change it if it isn't already set!
```

Comment:
So there is no uniqueness check? Excellent!

Reply:
Funnily enough, there is an index on the field but it isn't a unique index.